src/components/Games/Form.tsx

```diff
@@ -60,39 +84,57 @@ const Form: React.FC<Props> = ({ onSubmit }) => {
         }}
       >
         <Typography variant="h3" sx={{ my: 2, textAlign: "center" }}>
-          Register the Game
+          {editCard ? "Edit the Game" : "Register the Game"}
         </Typography>
 
         <TextField
           fullWidth
-          label="Name of the Game"
+          label={placeholders.name}
           variant="outlined"
-          value={name}
-          onChange={(e) => setName(e.target.value)}
+          value={form.watch("name")}
+          {...register("name", {
+            required: { value: true, message: "Name is required" },
+            maxLength: {
+              value: 30,
+              message: "Name cannot exceed 30 characters",
+            },
+            onChange: (e) => setValue("name", e.target.value),
+          })}
         />
+        <Typography sx={{ color: errorColor, fontSize: size }}>
+          {errors.name?.message}
+        </Typography>
```

Comment:
please check this simmilar (issue) https://github.com/proga-react-practice/xi-team-project/pull/3#discussion_r1598137947